index.html

```diff
@@ -10467,6 +10484,44 @@ <h3>Relationship Attributes</h3>
 		<h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<p>Below is an alphabetical list of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>states</a> and <a>properties</a> to be used by rich internet application authors. A detailed definition of each <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> state and <a>property</a> follows this compact list.</p>
 		<p id="index_state_prop" class="placeholder">Placeholder for index of states and properties</p>
+		<div class="property" id="aria-actions">
+			<pdef>aria-actions</pdef>
+			<div class="property-description">
+				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger behaviors or operations relevant to the current element.</p>
+				<p>The <code>aria-actions</code> attribute provides an element to other interactive elements which trigger actions on the current element. For example, in a web mail application, the currently selected email message may reference action elements such as the <em>message-specific</em> Reply, Forward, and Delete buttons. Actions are triggered by mainstream pointer events (click, tap) or by simulated pointer events from assistive technology.</p>
+				<ul>
+					<li>Authors MUST ensure that related action elements have a label.</li>
+					<li>Authors MUST ensure that related action elements accept a <code>click</code> event handler or default activation behavior.</li>
+					<li>Authors MUST ensure that related action events are not limited to a modality-specific activation such as the <code>keyup</code> or <code>touchend</code> event handlers.</li>
+					<li>Authors SHOULD ensure that related actions elements have subclass role of <rref>command</rref> such as <rref>button</rref> or <rref>link</rref>. [Todo: What about the "activatable" <rref>input</rref> subclass roles like <rref>checkbox</rref> and <rref>switch</rref><rref>? I think UAs could make those work by combining label and status: action name could be "uncheck 'subscribe to marketing emails'". -jc]</li>
+					<li>Authors SHOULD ensure that related actions elements are visible and activatable when the current element is focused by the user agent or assistive technology.</li>
+					<li>User Agents SHOULD use the label of each associated element to determine the name of the action in a platform accessibility API mapping.</li>
```

Comment:
Related to the previous comment, I left this "_to determine_ the name of the action" rather than "_as_ the name of the action" so that we may be able to include actionable inputs like checkbox and switch, not just subclasses of command.